docs/data/gridftp.md

```diff
@@ -111,6 +111,26 @@ All other configuration settings should be suitable for most purposes. However,
 
 ### Optional configuration
 
+### Modifying the limits
+
+Create file named `/etc/sysconfig/gridftp-hdfs`. An example configuration would be:
```

Comment:
When starting a section, we should provide context on "what" and "why". Stealing part of your sentence from below:

> To set a limit on the total or per-user number of transfers, set the following configuration:

The config example applies to gridftp-hdfs but what about regular gridftp or gridftp-xrootd? Let's make a table with column 1 listing all the gridftp flavors and column 2 listing the relevant config files